recipes/urlscan/meta.yaml

```
@@ -0,0 +1,42 @@
+{% set name = "urlscan" %}
+{% set version = "0.9.5" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/urlscan-{{ version }}.tar.gz
+  sha256: a2fb7513c8b6bf11d796d16353517b549f0070b08e2e9cf16ce482ab72e46c1f
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - python >=3.3
+    - urwid >=1.2.1
+
```

Adding this will make your package unix specific but still noarch.  Staged recipes will fail, 
But the resulting package will only be installable on unix. 

```suggestion
    - __unix
```